website/docs/r/kubernetes_cluster.html.markdown

```diff
@@ -714,6 +714,8 @@ A `web_app_routing` block supports the following:
 
  ---
 
+---
```

Comment:
there's now two dividers here?

Reply:
ha, the already existing divider has a space before it, updated